src/Widgets/PreferencesDialog.vala

```diff
@@ -19,6 +19,45 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
+/**
+ *  The purpose of this class is to easily create a new Switchbox for the
+ *  PreferencesDialog class.
+ */
+private class ENotes.PreferencesDialogSwitchBox {
+
```

Comment:
But you know... I like the idea. But i would remove it at least for this PR. If you still want, you can make another PR that changes all labes + switches to using this :) 

_Small diff is best diff_